### PR TITLE
Promise

### PR DESCRIPTION
**Pull request by dead-horse**

### CODE REVIEW

File: test/index.js [hunk 1]
```diff
@@ -67,4 +67,14 @@ describe('ready', function() {
       done();
     }, 10);
   });
-});
\ No newline at end of file
+});
```
Comment (fengmk2) [inline test/index.js:70]: add one more `yield this.ready()` test case

Comment (dead-horse) [inline test/index.js:70]: done

**Comment by fengmk2** [comment]
改名？`get-ready`?

**Comment by fengmk2** [comment]
or `ready2` ?

**Comment by hotoo** [comment]
jiuxu

**Comment by alsotang** [comment]
redy

2015-09-28 14:40 GMT+08:00 闲耘™ notifications@github.com:

**Comment by fengmk2** [comment]
haole

**Comment by hotoo** [comment]
haomei?

**Comment by fengmk2** [comment]
ping @dead-horse

**Comment by dead-horse** [comment]
get-ready +1

**Comment by fengmk2** [comment]
我发布一个版本

**Comment by fengmk2** [comment]
`get-ready@1.0.0`

**Comment by alsotang** [comment]
cool

2015-09-29 15:58 GMT+08:00 fengmk2 notifications@github.com:

**Pull request merged**